src/Database/Driver.php

```diff
@@ -539,11 +546,61 @@ public function getMaxAliasLength(): ?int
     /**
      * @inheritDoc
      */
-    public function setLogger(?LoggerInterface $logger): void
+    public function getLogger(): ?LoggerInterface
+    {
+        return $this->logger;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function setLogger(LoggerInterface $logger): void
     {
         $this->logger = $logger;
     }
 
+    /**
+     * Create logger instance.
+     *
+     * @param string|bool $log Log config.
+     * @return \Psr\Log\LoggerInterface
+     * @psalm-param string|true $log
+     */
+    protected function createLogger(string|bool $log): LoggerInterface
```

Comment:
I think `null` is better than `true` here.

Reply:
Done